src/cli/cli.cpp

```diff
@@ -781,6 +784,42 @@ void Interpreter::ProcessCoapSecure(int argc, char *argv[])
 
 #endif // OPENTHREAD_CONFIG_COAP_SECURE_API_ENABLE
 
+#if OPENTHREAD_CONFIG_PLATFORM_RADIO_COEX_METRICS_ENABLE
+void Interpreter::ProcessCoexMetrics(int argc, char *argv[])
+{
+    OT_UNUSED_VARIABLE(argc);
+    OT_UNUSED_VARIABLE(argv);
+
+    otRadioCoexMetrics metrics;
+    otError            error = otPlatRadioGetCoexMetrics(mInstance, &metrics);
+
+    SuccessOrExit(error);
+
+    mServer->OutputFormat("Stopped: %s\r\n", metrics.mStopped ? "true" : "false");
+    mServer->OutputFormat("NumGrantGlitch: %d\r\n", metrics.mNumGrantGlitch);
```

Comment:
`d` to `u`.

Reply:
If these are not to be scraped (hopefully), then a more readable format is preferred. E.g.:

```
Saturated: false
Grant glitches: xxx
Transmit metrics
    Requests: xxx
    Immediate grants: xxx
    Grant waits: xxx
    Grant waits activated: xxx
    Grant wait timeouts: xxx
    Grant deactivations during request: xxx
    .
    .
    .
Receive metrics
    Requests: xxx
     .
     .
     .
```

Reply:
Actually, I just realized the symmetry of the Tx and Rx stats, so a better format would be

```
                     Tx                    Rx
Requests:            xxx                   yyy
Immediate grants:    xxx                   yyy
.
.
.
```

Reply:
@smides-nest The Tx and Rx stats is asymmetrical. The Tx stats doesn't contain `NumTxGrantNone`. So it is better to follow your first suggestion.